### PR TITLE
Added Perl and CPANM installation guidelines using Perlbrew

### PR DESCRIPTION
**Pull request by henrikstranneheim**

### CODE REVIEW

File: perl/installation/installation.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+### Perl
+Install a specific perl version and use it as default.
+```Bash
+$ perlbrew install perl-5.26.0
+$ perlbrew switch perl-5.26.0
+```
+
+### Cpanm
+Is a lightweigth CPAN client, which facilitates installing CPAN perl modules. It is a good idea to install it together with Perlbrew to always make them available across all your perl installations.
```
Comment (jemten) [inline perl/installation/installation.md]: Are the modules really available across all the perl versions that you install via perlbrew?
My interpretation is that you install cpanm via perlbrew in order to make sure that cpanm is available across the different perl distributions. The point is to be able to have a different cpan library for each perl distribution. You can also create different cpan libraries to the same perl distribution and then switch between the different libraries. Then you get something like this when you run "perlbrew list" 
[anders.jemt@rastapopoulos MIP]$ perlbrew list
    perl-5.18.4
    perl-5.18.4@mip_18
    perl-5.26.1
    perl-5.26.1@cpanfile
\* perl-5.26.1@mip_develop

File: perl/installation/installation.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+1. Log out and in again to initilize a fresh shell
+1. Follow the instructions at [Perlbrew] for Installing Perlbrew using curl or wget.
+1. Append the following piece of code to the end of your `~/.bash_profile`:
+```Bash
+    source ~/perl5/perlbrew/etc/bashrc
+```
+Start a new shell and perlbrew should be up and fully functional from there.
+
+### Perl
+Install a specific perl version and use it as default.
```
[the remark this answers is not in the document]
Comment (henrikstranneheim) [reply perl/installation/installation.md]: Good point! We will add that to MIPs README. This is more general for Perl. And soon they will align (together with the sun and the moon) 😄

File: perl/installation/installation.md (hunk 1 above)
Comment (henrikstranneheim) [reply perl/installation/installation.md]: Ah, the point was just that they tag along when you switch perl version. I will try to make it more clear

Comment (henrikstranneheim) [reply perl/installation/installation.md]: Also it would be great if you could add the command to have multiple CPANM libs per perl version.

**Comment by henrikstranneheim** [comment]
@emiliaol I will go ahead and merge this. You can read about it when you have time.

**Comment by emiliaol** [comment]
Yeah, sorry for missing this but I'll make sure to read it through and if I find anything I'll let you know.

**Pull request merged**